### ISSUE TITLE
Resource

### ISSUE THREAD

**Issue by xxggt** [LancerComet/Muso#2]
I need an adaptation of this to enable reading of both manga and webtoons. Currently, it works well for manga, but it's impossible to navigate when it comes to webtoons.

**Comment by LancerComet** [comment]
Have you tried vertical scrolling mode? Does it meet your needs?

**Comment by xxggt** [comment]
Yes, I tried, but the scrolling is completely off, making it impossible to read

**Comment by LancerComet** [comment]
Hum, may I ask you that what kind of input device you are using? Touchscreen and mousewheel should work properly, but I don't know how it works on Macbook using touchpad 🤔
